packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -245,7 +275,8 @@ export class ChildLogger extends TelemetryLogger {
     public static create(
         baseLogger?: ITelemetryBaseLogger,
         namespace?: string,
-        properties?: ITelemetryLoggerPropertyBags): TelemetryLogger {
+        properties?: ITelemetryLoggerPropertyBags,
+        supportsTags?: true | undefined): TelemetryLogger {
```

Comment:
It does not look like supportsTags should be here, instead baseLogger.supportsTags  should be used by ChildLogger itself